src/config/gmData.js

```diff
@@ -0,0 +1,109 @@
+export default {
```

Comment:
Over data structure looks great!

I'm now realizing that these data files would be great places for Typescript, so we can enforce more strict compile time rules about formatting (i.e. the comment you wrote below about `month` and `day`. Would love to hear @mattxwang's take on this.

Perhaps out of scope for this PR, but you could also consider adding some smoke tests for the data. (i.e. regex checking length of strings, formatting etc).

Reply:
I ended up deciding against that since I wasn't sure what the error message or error handling situation for that should be - since right now giving it an invalid input makes date NAN - which makes it obvious along with the comment what the error is.

Do you think I should still do it? If so what kind of error message would you say make sense?

Reply:
I think it's probably more work than its worth and I'd rather ship this PR as is for the following reasons
- The data model is pretty well documented in comments and in this PR
- This file will likely see pretty light use, given we only have 2 GMs every year

Final call is up to you.

Reply:
This makes sense! 
Once it's done being reviewed I'll merge it into main then.